pages/osx/log.md

```diff
@@ -0,0 +1,20 @@
+# log
+
+> View, export, and configure logging systems.
+> More information: <https://www.dssw.co.uk/reference/log.html>.
+
+- Stream live system logs:
+
+`log stream`
+
+- Stream logs sent to syslog from the process with a specific PID:
+
+`log stream --process {{235}}`
```

Comment:
```suggestion
`log stream --process {{PID}}`
```

Reply:
the placeholders should be in `snake_case`.

Reply:
PID is an acronym, therefore it should be in caps (Process ID)

Reply:
You can use `{{process_id}}` to be more friendly for new users.

Reply:
It is an acronym, but in this case it should be lowercase regardless

Reply:
I like process_id

my suggestion has been edited